### DISCUSSION TITLE
一张图实现柱状图在x轴上方与下方并在上面将二者的差距用面积图实现

### DISCUSSION THREAD

**Discussion by shengminhe** [antvis/G2#5115] [Q&A]
### Discussed in https://github.com/antvis/G2/discussions/5108

<div type='discussions-op-text'>

<sup>Originally posted by **shengminhe** May 29, 2023</sup>
阿里云效中缺陷收敛：
![微信图片_20230529122956](https://github.com/antvis/G2/assets/59502108/8a93f214-4e2b-4463-87ba-a7ebbbc1414e)
一张图实现柱状图在x轴上方与下方并在上面将二者的差距用面积图实现</div>

**Comment by hustcc** [comment]
重复：https://github.com/antvis/G2/discussions/5108
